ApiSurface/VersionFile.fs

```diff
@@ -2,34 +2,44 @@ namespace ApiSurface
 
 open System.IO
 open System.IO.Abstractions
-open Newtonsoft.Json
+open System.Text.Json
 open System.Reflection
+open System.Text.Json.Serialization
 
 /// A record representing the layout of a version.json file, e.g. as consumed by NerdBank.GitVersioning.
 type VersionFile =
     {
-        /// The version number (e.g. "1.0.1")
-        [<JsonProperty("version")>]
+        /// The version number (e.g. "1.0")
+        [<JsonPropertyName "version">]
```

Comment:
Can remove these attributes and instead use https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonnamingpolicy.camelcase?view=net-7.0